src/tmx/TmxUtils/src/kafka/kafka_consumer_worker.cpp

```diff
@@ -88,6 +94,7 @@ namespace tmx::utils
     void kafka_consumer_worker::stop()
     {
         _run = false;
+        _consumer->close();
```

Comment:
Can we leave an inline comment that this close call removes the consumer from the consumer group in the kafka broker.

Reply:
Added